justfile

```
@@ -234,12 +234,17 @@ addnoun noun:
     exit 0
   fi
 
-  if [[ "{{noun}}" =~ ^[A-Z] ]]; then
-    echo "{{noun}}/M" >> $DICT_FILE
-  else
-    echo "{{noun}}/SM" >> $DICT_FILE
+  # 'M', possessive -'s suffix for both common and proper nouns
+  flags='M'
+
+  # 'S', plural -(e)s suffix only for common nouns (not capitalized)
+  if [[ "{{noun}}" =~ ^[a-z] ]]; then
+    flags+='S'
   fi
```

```suggestion
  # detect nouns (not capitalized)
  if [[ "{{noun}}" =~ ^[a-z] ]]; then
   # 'S', plural -(e)s suffix
    flags+='S'
   fi
```

Maybe something like this. You can rephrase of course, but you get the point I think